package.py

```diff
@@ -1085,13 +1089,13 @@ def copy_file_to_target(file, temp_dir):
         pyproject_target_file = copy_file_to_target(pyproject_file, temp_dir)
 
         if os.path.isfile(poetry_lock_file):
-            log.info("Using poetry lock file: %s", poetry_lock_file)
+            log.info("Using poetry.lock file: %s", poetry_lock_file)
```

Comment:
This was a minor/superficial edit as most of the Poetry documentation refers to the lock file directly as `poetry.lock`. yes, the output should still be the path to the `poetry.lock` itself. In the end, it can be removed if undesired.

Reply:
Ok, let's keep this as is.